integration-tests/cli/csv-parser-price.it.js

```diff
@@ -119,6 +116,26 @@ describe('CSV and CLI Tests', () => {
         },
       )
     })
+
+    test('should log errors to stderr and log file', (done) => {
```

Comment:
Are you logging to `stderr` and the logFile at the same time? I thought it's one or the other